functions_framework/lib/serve.dart

```diff
@@ -35,9 +35,12 @@ export 'src/function_target.dart'
 /// If there are no configuration errors, this function will not return until
 /// the process has received signal [ProcessSignal.sigterm] or
 /// [ProcessSignal.sigint].
-Future<void> serve(List<String> args, Set<FunctionTarget> targets) async {
+Future<void> serve(
+  List<String> args,
+  FunctionTarget Function(String) functionForName,
```

Comment:
`functionForName` sounds a little funny to me. Is this the `functionTarget`? Or `functionTargetName`?

Reply:
It's meant to return the `FunctionTarget` instance for a given (`String`) target name.

Suggestions welcome!

Reply:
PTAl